ajax/libs/angular-gridster/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-gridster",
+  "description": "An implementation of gridster-like widgets for Angular JS. This directive gives you gridster behavior",
+  "filename": "angular-gridster.min.js",
+  "keywords": [
+    "angular",
+    "widget",
+    "gridster-like",
+    "angular directives"
+  ],
+  "author": "https://github.com/ManifestWebDesign/angular-gridster/graphs/contributors",
+  "homepage": "http://manifestwebdesign.github.io/angular-gridster",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ManifestWebDesign/angular-gridster.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git:github.com//ManifestWebDesign/angular-gridster.git",
```

Comment:
Let's modify to `"target": "git://github.com/ManifestWebDesign/angular-gridster.git",`
Because this link is available.
Thank you.

Reply:
The origin url is not a valid format